XpringKit/XRPPayment.swift

```diff
@@ -5,11 +5,24 @@ import Foundation
 public struct XRPPayment: Equatable {
   // TODO(keefertaylor): Modify this object to use X-Address format.
 
+  /// The amount of currency to deliver.
   public let amount: XRPCurrencyAmount
+
+  /// The unique address of the account receiving the payment.
   public let destination: Address
+
+  /// (Optional) Arbitrary tag that identifies the reason for the payment.
```

Comment:
Tags don't identify the reason for the payment, they indicate which user to send the payment to.

For example, Coinbase probably has one (or a handful) of actual XRP addresses, but each Coinbase user gets a tag number. So you send the XRP to Coinbase, and then Coinbase uses the tag number to credit the respective user inside their system.

Reply:
I've updated this comment across all three SDKs... Check out the new version?  It's based on the documentation on xrpl.org docs here (under destinationTag): https://xrpl.org/payment.html.  Do you think that works, or should I entirely get rid of the bit about reasons?

Reply:
I think it's reasonable to use that language if that's what `xrpl.org` says.